packages/core/src/browser/style/tree.css

```diff
@@ -108,6 +108,18 @@
     text-align: center;
 }
 
+.theia-TreeNode .theia-TreeNodeTailInlineCommand  {
+    visibility: hidden;
+}
+
+.theia-TreeNode:hover .theia-TreeNodeTailInlineCommand  {
+    visibility: visible;
+}
+
+.theia-TreeNode.theia-mod-selected .theia-TreeNodeTailInlineCommand {
```

Comment:
Tree decorator styles should go the `tree-decorators.css`, and the new constants should go under [`Styles`](https://github.com/theia-ide/theia/blob/756bc5d182c10b1af46045f3211c302f20192a29/packages/core/src/browser/tree/tree-decorator.ts#L171).

Reply:
done